__fixtures__/test-project/package.json

```diff
@@ -7,7 +7,8 @@
     ]
   },
   "devDependencies": {
-    "@redwoodjs/core": "8.0.0-canary.150"
+    "@redwoodjs/core": "8.0.0-canary.150",
+    "@redwoodjs/studio": "11"
```

Comment:
If we remove, when `yarn rw studio` is run for the first time, it will add here -- so just leaving in by default.